### PR TITLE
regexp not regex

### PR DESCRIPTION
**Pull request by calellowitz**
@millerdev 
typo in last PR. this one runs

### CODE REVIEW

File: ansible/roles/ssh/tasks/main.yml [hunk 1]
```diff
@@ -17,9 +17,9 @@
 - name: Ensure challenge response access is disallowed
   lineinfile:
     dest: /etc/ssh/sshd_config
-    regex: "^ChallengeResponseAuthentication"
+    regexp: "^ChallengeResponseAuthentication"
     line: "ChallengeResponseAuthentication no"
-    state: absent
+    state: present
```
Comment (calellowitz) [inline ansible/roles/ssh/tasks/main.yml]: changed from removing `yes` line to adding explicit `no` line, and forgot to update

**Comment by calellowitz** [comment]
Now deployed to india machines and confirmed that only `publickey` shows up in the list of authentications

**Comment by millerdev** [comment]
heh, just hit this while deploying new riak nodes

**Comment by millerdev** [comment]
👍 on ✅

**Pull request merged**